ibm/service/scc/resource_ibm_scc_control_library.go

```diff
@@ -876,6 +887,17 @@ func resourceIbmSccControlLibraryControlsInControlLibToMap(model *securityandcom
 	return modelMap, nil
 }
 
+// using the assessment_id for comparison
+func compareAssessmentSetFunc(v interface{}) int {
+	m := v.(map[string]interface{})
```

Comment:
Can you add some checks to check len of map and element of map not nil to eliminate any nil pointer exceptions

Reply:
In schema assessment_id is defined as optional if user define 
assessments = {
assessment_method = "XXXXX"
}